openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -5,6 +5,7 @@
 {{- end }}
 
 log_config_append = /etc/nova/logging.ini
+logging_context_format_string = %(process)d %(levelname)s %(name)s [%(request_id)s g%(global_request_id)s %(user_identity)s] %(instance)s%(message)s
```

Comment:
Do we need an additional ` ` bewetween instance and message or is this like before? Can you maybe point to where you got the string from? I assume you just extended it and it would be good to know for reference later on.

Reply:
The original comes from here: https://github.com/openstack/oslo.log/blob/master/oslo_log/_options.py#L148-L150, it also has no white-space between `$(instance)s` and `$(message)s`
(Well, I got it from here: https://github.com/sapcc/helm-charts/blob/master/openstack/keystone/templates/etc/_keystone.conf.tpl#L9, but that is likely from the Oslo.log defaults).

Reply:
Can we reference this in the commit-message, please? :)